Patterns/English/English-DateTime.yaml

```diff
@@ -516,7 +516,7 @@ UnspecificEndOfRegex: !simpleRegex
 UnspecificEndOfRangeRegex: !simpleRegex
```

Comment:
Please remove the .github/Recognizers-Text - Shortcut.lnk file from the PR.

Reply:
removed shortcut link from PR